pycbc/events/single.py

```diff
@@ -158,43 +198,62 @@ def check(self, trigs, data_reader):
         # Apply cuts to trigs before clustering
         # Cut on snr so that triggers which could not reach newsnr
         # threshold do not have newsnr calculated
+        if 'psd_var_val' in trigs:
+            # We should apply the PSD variation rescaling, as this can
+            # re-weight the SNR to be above SNR
+            trig_chisq = trigs['chisq'] / trigs['psd_var_val']
+            trig_snr = trigs['snr'] / (trigs['psd_var_val'] ** 0.5)
+        else:
+            trig_chisq = trigs['chisq']
+            trig_snr = trigs['snr']
+
         valid_idx = (trigs['template_duration'] >
                      self.thresholds['duration']) & \
-                    (trigs['chisq'] <
+                    (trig_chisq <
                      self.thresholds['reduced_chisq']) & \
-                    (trigs['snr'] >
+                    (trig_snr >
```

Comment:
This seems to effectively change the meaning of `--reduced-chisq-threshold` and `--newsnr-threshold` in case the PSD var statistic is used, i.e. "reduced chisq" and "newsnr" will no longer mean what people historically think they mean. Should we rename the two options then?